packages/react-components/src/molecules/MembersList.tsx

```diff
@@ -75,11 +75,15 @@ interface MembersListProps {
   >;
   singleColumn?: boolean;
   readonly overrideNameStyles?: SerializedStyles;
+  readonly overrideUserRoute?: ({ userId }: { userId: string }) => {
```

Comment:
can we just call it user route and default it to the crn one?

Reply:
done